### ISSUE TITLE
Pokebattler import is partially failing

### ISSUE THREAD

**Issue by Artanicus** [pokepark/PokemonRaidBot#248]
Pulling relevant bits from @Ninjasoturi on TG:

```
[2021-08-13 21:45:46.1402][111740] [pokebattler.php:135] * Skipping raid boss AMPHAROS_MEGA since it has no id, it's likely in the future!
```
Looks like our logic is faulty.

**Comment by Artanicus** [comment]
Theres a few potential issues I see:
- RAID_LEVEL_6 has been retired and replaced by RAID_LEVEL_MEGA
- The breakingNews section could be used for prefilling upcoming bosses, has valid IDs and timestamps.
- RAID_LEVEL_5 seems otherwise fine but for some reason doesn't contain id's anymore. 

-> Best bet is migrating to the breakingNews dictionary coupled with timestamp checks to check which ones are active

**Comment by Artanicus** [comment]
Fixed in #249
